.nuspec/Xamarin.Forms.MultiTargeting.targets

```diff
@@ -2,22 +2,32 @@
 	<ItemGroup Condition="$(TargetFramework.StartsWith('Xamarin.iOS')) != true ">
 		<Compile Remove="**\*.iOS.cs" />
 		<None Include="**\*.iOS.cs" />
+    <Compile Remove="**\iOS\*.cs" />
```

Comment:
Right now for our platform folders we have directories named after the platforms and then files in those folders with "extensions" 

These allow for a project to separate by folder name if they want to